README.md

```diff
@@ -616,7 +617,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Franklin Ohaegbulam](https://github.com/frankiefab100)
   - [Frazier Odhiambo](https://github.com/frazie)
   - [Furqan Abid](https://github.com/heyyfurqan)
-  - [Farhan Ali](https://github.com/Farhan-Ali2002)
+  
```

Comment:
Hey @Farhan-Ali2002, Could you please remove this extra space? Apart from that, everything looks good to me.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix